coalib/bearlib/aspects/Formatting.py

```diff
@@ -0,0 +1,395 @@
+from coalib.bearlib.aspects import Root, Taste
+
+
+@Root.subaspect
+class Formatting:
+    """
+    Formatting refers to the visual appearance of you source code.
+    """
+    class docs:
+        example = """
+        # Here is an example of python code with lots of
+        # formatting issues including: trailing spaces, missing spaces
+        # around operators, incorrect indentation etc...
+
+        z = 'hello '+'world'
+        .....def f  (   a ) :
+        ........pass
+        """
+        example_language = 'python'
+        importance_reason = """
+        Your coding style (the of rules or guidelines used when writing the
+        source code) can drastically affect the readability, and
+        maintainability of your program and might as well introduce bugs.
+        """
+        fix_suggestions = """
+        Defining a clearly and thoughtful coding style (based on the available
+        ones given the programming language in use) and strictly respect it or
+        apply it through out the implementation of your project.
+        """
+
+
+@Formatting.subaspect
+class Length:
+    """
+    The `Length` here refers to:
```

Comment:
This needs to be a single line summary ending in a full stop.

If that one line summary isnt sufficient, more precise info should be added after a blank line.